src/components/Menu/styles.module.css

```diff
@@ -1,23 +1,43 @@
-.nav {
-  position: absolute;
-}
-
-.nav li {
-  cursor: pointer;
-  list-style-type: none;
-  width: 120px;
-  color: white;
-  background-color: #3a77d7;
-  border-radius: 2px;
-  margin-right: 5px;
-  padding-top: 10px;
-  padding-bottom: 10px;
-}
-
-.ul {
-  display: flex;
-}
-
-.nav a {
-  text-decoration: none;
+.link{
+    text-decoration: none;
+    padding: 17px;
+    border-radius: 3px;
+    color: #e7b829e3;
+}
```

Comment:
For consistency and readability, please leave a whitespace before the opening brackets and also an empty line after each set of rules.
For example, instead of this:

```
.link{
    text-decoration: none;
    padding: 17px;
    border-radius: 3px;
    color: #e7b829e3;
}
.activeLink{
    background-color: white;
    color: blue;
}
.buttonBar{
    display: flex;
}
```

You may prefer adding the styles like this:
```
.link {
    text-decoration: none;
    padding: 17px;
    border-radius: 3px;
    color: #e7b829e3;
}

.activeLink {
    background-color: white;
    color: blue;
}

.buttonBar {
    display: flex;
}
```

Please update all this file's content appropiately.